src/qt/guiutil.cpp

```diff
@@ -732,6 +734,16 @@ QString formatDurationStr(std::chrono::seconds dur)
     return strList.join(" ");
 }
 
+QString FormatPeerAge(std::chrono::seconds time_connected)
```

Comment:
abf93ddcdf5802f9b76e747a7669bc65bbda92cc

I think this could be idempotent `QString FormatAge(std::chrono::seconds age)`, to avoid `GetTime`.

Note that `PeerTableModel` has a timer to refresh the data, just call `GetTime` once there.

Reply:
@promag I agree this would be an improvement and am looking into it, but am not sure how to integrate GetTime into the QTimer code. Pointers welcome.